pkg/controller/cheinstallation/che.go

```diff
@@ -13,9 +13,9 @@ import (
 
 const (
 	// InstallationName the name of the CheInstallation resource (cluster-scoped)
-	InstallationName = "toolchain-che-installation"
+	InstallationName = "toolchain-workspaces-installation"
 	// Namespace the namespace in which the OLM OperatorGroup and Subscription resources will be created
-	Namespace = "toolchain-che"
+	Namespace = "toolchain-workspaces"
```

Comment:
'crw' is another option here .. `toolchain-crw`.

Reply:
@nurali-techie we agreed on this name - for more information see https://jira.coreos.com/browse/CRT-364?focusedCommentId=154212&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-154212 and also discussion in slack channel https://coreos.slack.com/archives/CHK0J6HT6/p1576162588107500?thread_ts=1576151960.097700&cid=CHK0J6HT6